### DISCUSSION TITLE
Including files in build

### DISCUSSION THREAD

**Discussion by ElYaiko** [nuxt/nuxt#15988] [Questions]
Im trying to include other files in the build, already tried rollup options and nitro options s: I want to include other files on the .output/server file that are not being included in the build.
For example server/scripts/s1.ts should be included in the .output/server/chunks as .mjs, anyone knows how to do this? 🙏

**Comment by gregg-cbs** [comment]
I have this same question.

I have a json file with my data in it that I read and write to, i want this file to be part of the server only but I cant seem to find a place to put it that it gets copied over in the build - because i dont import the file, i only reference it with the fileReader, vite doesnt know to include this file.

So i tried putting the file in the 'public' folder and this works on my windows machine but I get an error on my linux server saying:
"ENOENT: no such file or directory, open 'public/data/orders.json'"
^ seems my windows machine proxies the 'public' folder but on linux it doesnt

Im not sure how to get around this. If i could copy the file into the server folder that would solve this problem.

**Comment by gregg-cbs** [comment]
@ElYaiko 
I managed to do this by adding a small bash script.. you could write a file and run that for more complicated stuff but this is what I did for my one file:

 #package.json
```json
"build": "nuxt build && cp data/orders.json .output/server",
```

**Comment by mehdirayan** [comment]
I have the same issue
I'm using serialport in my project. In dev mode all works fine.
but when I build the project, the binding-cpp folder in not created the server project.
I've tried many options without success

**Reply by gregg-cbs** [reply]
Why i ditched Nuxt for Nextjs. My life is good now.
